### PR TITLE
Feat/communities

### PR DESCRIPTION
**Pull request by anyulled**
Including:

1. step4ward
2. Eclipse foundation.
3. Apache foundation.

As supporting communities

### CODE REVIEW

**Review by icougil — CHANGES_REQUESTED** [review]
I think the link for the Apache foundation is not correct.
Out of this, everything looks ok for me

**Comment by anyulled** [comment]
> I think the link for the Apache foundation is not correct. Out of this, everything looks ok for me

It's the same one we used last year. https://www.jbcnconf.com/2022/foss.html

**Comment by icougil** [comment]
I don't know, but the link should be https://apache.org/

File: src/views/Home/components/Sponsors/SponsorsData.ts [hunk 1]
```diff
@@ -136,6 +136,21 @@ export const sponsors: Sponsors = {
...
     },
+    {
+      name: "step4ward",
+      image: "images/sponsors/step4ward.png",
+      website: "https://bit.ly/step4wardhome",
+    },
+    {
+      name: "Apache foundation",
+      image: "images/sponsors/apache-foundation.jpeg",
+      website: "https://www.apache.com/",
```
Comment (icougil) [inline src/views/Home/components/Sponsors/SponsorsData.ts:147]:
```suggestion
      website: "https://www.apache.org/",
```

**Review by icougil — CHANGES_REQUESTED** [review]

**Review by icougil — APPROVED** [review]

**Pull request merged**